docs/rum_collection/crash_reporting.md

```diff
@@ -25,16 +25,30 @@ Enable iOS Crash Reporting and Error Tracking to get comprehensive crash reports
 
 ### Add crash reporting 
 
-If you have not set up the SDK yet, follow the [in-app setup instructions][1] or refer to the [iOS RUM setup documentation][2]. 
-
-Add the `DatadogCrashReporting` dependency to your project. For CocoaPods, use `pod 'DatadogSDKCrashReporting'`. For SPM and Carthage, `DatadogCrashReporting` is available with `dd-sdk-ios`.
-
-| Package manager            | Installation method                                                                         |
-|----------------------------|-------------------------------------------------------|
-| CocoaPods                  | Use `pod 'DatadogSDKCrashReporting'`                      |
-| Swift Package Manager      | Link the `DatadogCrashReporting` module                   |
-| Carthage                   | Use `DatadogCrashReporting.xcframework`               |
-
+If you have not set up the SDK yet, follow the [in-app setup instructions][1] or refer to the [iOS RUM setup documentation][2].
+
+#### Dependency Manager
+{{< tabs >}}
+{{% tab "CocoaPods" %}}
+Add `DatadogSDKCrashReporting` to your `Podfile`:
+```ruby
+platform :ios, '11.0'
+use_frameworks!
+
+target 'App' do
+  pod 'DatadogSDKCrashReporting'
+end
+```
+{{% /tab %}}
+{{% tab "Swift Package Manager" %}}
+Add the Package at `https://github.com/DataDog/dd-sdk-ios` then link the `DatadogCrashReporting` library to your application target. 
+> If your already linking `Datadog` or `DatadogStatic` library, replace it with `DatadogCrashReporting`.
```

Comment:
How can I mark this line as 'important', highlighting somehow?

Reply:
```suggestion

**Note:** If you link to `Datadog` or the `DatadogStatic` library, replace it with `DatadogCrashReporting`.
```